src/Underworld.ts

```diff
@@ -147,10 +151,15 @@ export default class Underworld {
   pie: PieClient | IHostApp;
   // The index of the level the players are on
   levelIndex: number = -1;
+  isTutorialRun: boolean = false;
   wave: number = 0;
   // for serializing random: prng
   RNGState?: SeedrandomState;
   turn_phase: turn_phase = turn_phase.Stalled;
+  // The order in which units will take their turn
+  subTypesTurnOrder: UnitSubType[][] = [[UnitSubType.RANGED_LOS, UnitSubType.RANGED_RADIUS], [UnitSubType.SUPPORT_CLASS], [UnitSubType.MELEE], [UnitSubType.SPECIAL_LOS], [UnitSubType.DOODAD]];
```

Comment:
Why do Doodads have a turn?

Reply:
This is how it was implemented previously in executeNPCTurn() so I left it :P

I think it's good though. If any future doodads ever have some kind of action to take, this would allow that to happen. There should be no downside to it since they'll be skipped if no doodads can do anything.